.github/workflows/yocto-build-deploy.yml

```diff
@@ -982,18 +981,19 @@ jobs:
 
     env:
       BALENARC_BALENA_URL: ${{ vars.BALENA_HOST || vars.BALENARC_BALENA_URL || 'balena-cloud.com' }}
-      DEPLOY_PATH: ${{ github.workspace }}/deploy
       HOSTAPP_ORG: ${{ vars.HOSTAPP_ORG || 'balena_os' }}
+      DEPLOY_PATH: ${{ github.workspace }}/deploy
 
     steps:
 
       # Use gh to download the build artifacts from the build job
       # https://github.com/actions/download-artifact/issues/396#issuecomment-2796144940
       # https://cli.github.com/manual/gh_run_download
       # https://cli.github.com/manual/gh_help_environment
-      - name: Download build artifacts
+      - name: Download hostapp artifacts
```

Comment:
Nit: Given that the hostapp artifacts are a superset of the testing artifacts, do we gain some speed by taking the testing artfiacts set out of the hostapp artifacts now, instead of later - and then downloading the testing artifacts as well as hostapp artifacts in the "deploy hostapp" job?

It might not be worth the gains from not uploading the same thing twice , so just a question